src/layouts/Home.astro

```diff
@@ -176,3 +177,23 @@ import SponsorRow from "@parts/sponsors/SponsorRow.astro";
 	</div>
 
 </HeroContent>
+
+<script>
+	import { toast } from 'bulma-toast';
```

Comment:
Also added at the top of this file without any changes between them. This feels like something is extra in one of the two spots, but I'm not knowledgeable enough with Astro to tell for sure